src/app/pages/OriginsLaunchpad/index.tsx

```diff
@@ -4,15 +4,13 @@ import { useTranslation } from 'react-i18next';
 import { translations } from 'locales/i18n';
 import { Header } from 'app/components/Header';
 import { Footer } from 'app/components/Footer';
-import { Dashboard } from './pages/Dashboard';
+// import { Dashboard } from './pages/Dashboard';
 import { SalesDay } from './pages/SalesDay';
 /* undo once Fish contract has active sale tier reset to 0 */
```

Comment:
This comment is not true anymore and you don't need the import below it as well.